lib/utils/translateResults.js

```diff
@@ -36,12 +36,12 @@ const transformAllMarkdowns = input => {
 
 /**
  * Composes output text from result object with title and description.
- * @param {{title: string, description: string}} result
+ * @param {{title: {type: any, value: string}, description: {type: any, value: string}}} result
  * @returns {string} - composed text
  */
-const titleDescToText = result => result.description
-	? `${result.title}\n${result.description}`
-	: `${result.title}`;
+const titleDescToText = result => result.description && result.description.value
```

Comment:
@Noorama please, use `toText` function from @suitest/smst-to-text package for proper translating title and description